AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+|ControlSeverity|string|The severity of the control.|Critical|
+|FeatureName|string|Type of feature this control belongs to.|Storage|
+|ControlSettings|string|Settings specific to the control to be provided for scan.|{"ApplicableOsTypes": ["Windows"]}|
+|Tags|string|Labels that denote the control to be of specfic type and belongs to specific domain.|["Baseline", "SDL", "TCP","Automated" ] |
+|Enabled|bool|Field to indicate whether the control is enabled or disabled.|true|
+|PolicyDefinitionGuid|string|The policy definition guid.|0x0xxx00-x0x0-000x-xx0x-0000x0000000|
+|ResourceType|string|The type of resources this control scan.|"Microsoft.Storage/storageAccounts"|
+|Remediation|string|Steps to remediate control.|For remediation using PowerShell commands, run command: 'Set-AzStorageAccount -ResourceGroupName <RGName> -Name <StorageAccountName> -EnableHttpsTrafficOnly $true'. Run 'Get-Help Set-AzStorageAccount -full' for more help.|
+|ControlScanSource|string|The control scan source.|Reader|
+|AssessmentProperties|string|The assessment properties.|{"AssessmentNames": ["000000xx-0xxx-0x00-0x00-00x00x0000x0"]}|
+|CustomPolicyProperties|string|The custom policy properties.|{"PolicyDefinitionIds": ["/providers/Microsoft.Authorization/policyDefinitions/000xx000-xx0x-0000-00dd-000000xx000x"]},|
+|Automated|string|Whether the control is manual or automated.|Yes|
+
+___
+
+### **Control result data**
+
+|Field Name|Field Type|Description|Sample Value|
+|--|--|--|--|
+|JobId|int|JobID for scan (YYYYMMDD).|20230414|
+|subscriptionid|string|The subscription ID.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|ResourceGroupName|string|The resource group name.|TestRG|
+|ResourceName|string|The resource name.|storageaccount|
+|ResourceType|string|The resource type.|Microsoft.Storage/storageAccounts|
+|ResourceId|string|The resource ID.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/resourceGroups/sampleresourcegroup/providersMicrosoft.Storage/storageAccounts/resourcename|
+|ControlName|string|The control name.|Azure_Storage_DP_Encrypt_In_Transit|
+|VerificationResult|string|It indicates control status.|Passed|
+|ScannedOn|datetime|Control scanned date and time.|Fr, 14 Apr 2023 08:39:24 GMT|
+|StatusReason|string|Reason associated with control status.|Secure transfer is enabled for the storage account.|
+|ScannedBy|string|Scanned by.|null|
+|ScanSource|string|The scan source of control.|Reader|
+|AssessmentName|string|Name of the assessment evulated by control.|00x0x00x-x000-0xxx-x0x0-000x000x00x0|
+|CustomField1|string|This field contains custom data.|This field contains custom data.|
+|CustomField2|string|This field contains custom data.|This field contains custom data.|
+|CustomField3|string|This field contains custom data.|This field contains custom data.|
+|CustomField4|string|This field contains custom data.| field contains custom data.|
```

Comment:
"This" field contains custom data - typo?

Reply:
Yes. Corrected it.